mmstudio/src/main/java/org/micromanager/internal/positionlist/TileCreatorDlg.java

```diff
@@ -364,6 +376,13 @@ public void actionPerformed(ActionEvent arg0) {
       studio.events().registerForEvents(this);
    }
 
+   @Subscribe
+   public void stuttingDown(ShutdownCommencingEvent se) {
```

Comment:
Sorry, I do not understand the question.  Does this refer to the "registerForEvent", or to the "stuttingDown' (sic!) function?  The latter is needed to close the window on shutdown. Maybe you refer to something else?